### ISSUE TITLE
i want to solve  some ODEs and system of ODEs.  is this model able to work for my problem?

### ISSUE THREAD

**Issue by ehtisham409** [lululxvi/deepxde#12]

**Comment by lululxvi** [comment]
Yes. There are examples of ODE system, see ode_system.py for forward problem, and Lorenz_inverse.py for inverse problem.

**Comment by richieakka** [comment]
Hi @lululxvi, Thanks for sharing this wonderful Library Deep XDE. I am interested in solving a first order logistic DE with Initial Condition. The equation is :
dX/dt=aX (1-X/K) with the IC X(0)=200 and later for another case X(0)=1200. My predicted solution does not match my exact solution. Kindly Help…Can we use Deep XDE, if we don’t have BC and only IC. My Modified code for the same is :

a=1 #0.490
N_star = 1000 #17015

def ode_system(x, y):
    r = y[:, 0:]
    #d_r_over_d_t = dde.grad.jacobian(y, x, i=0)
    d_r_over_d_t = tf.gradients(r, x)[0]
    return [
        d_r_over_d_t - a*r*(1-r/N_star)
    ]

def boundary(_, on_initial):
    return on_initial
 
geom = dde.geometry.TimeDomain(0, 5) 
ic1 = dde.IC(geom, lambda X: 1, boundary, component=0)
#ic1 = dde.IC(geom, func, lambda _, on_initial: on_initial)
data = dde.data.PDE(geom, ode_system, [ic1], 350, 2, num_test=100) #change here, 4000 traininng points

layer_size = [1] + [20] * 3 + [1] # change here, layers are made 5 instaed of 3.
activation = "tanh" #"relu" #"sigmoid" #"tanh"
initializer = "Glorot uniform"
net = dde.maps.FNN(layer_size, activation, initializer)

model = dde.Model(data, net)
model.compile("adam", lr=0.001)
#model.compile("adam", lr=0.001, metrics=["l2 relative error"])
losshistory, train_state = model.train(epochs=10000)

dde.saveplot(losshistory, train_state, issave=True, isplot=True)

t = np.linspace(0, 5, 50).reshape(50, 1)
y = model.predict(t)

plt.xlabel('t')
plt.plot(t, y[:, 0:1])
plt.show()
--------------------
Please help us in getting it right.

Thanks
Richa Gupta

**Comment by lululxvi** [comment]
ODE system can be solve in DeepXDE. Here are some examples: https://deepxde.readthedocs.io/en/latest/demos/forward.html#odes

One issue in your problem is that the IC has a very large value, and you need to normalize it. See "Q: I failed to train the network or get the right solution, e.g., the training loss is large." in FAQ https://deepxde.readthedocs.io/en/latest/user/faq.html
